packages/web-components/fast-components/src/skeleton/fixtures/base.html

```diff
@@ -0,0 +1,169 @@
+<fast-design-system-provider use-defaults>
+    <style>
+        .card-placeholder {
+            background-color: #fff;
+            padding: 20px;
+            width: 500px;
+            border-radius: 4px;
+        }
+    </style>
+
+    <h1>Skeleton</h1>
+
+    <h4>Used as element blocks</h4>
+    <div class="card-placeholder">
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                width: 50px;
+                height: 50px;
+            "
+            shape="circle"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                margin-top: 10px;
+                height: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
+                border-radius: 4px;
+                width: 75px;
+                height: 30px;
+                margin-top: 20px;
+                margin-bottom: 10px;
+            "
+            shape="rect"
+        ></fluent-skeleton>
+    </div>
+
+    <h4>Used as element blocks with shimmer effect element</h4>
+    <div class="card-placeholder">
+        <fluent-skeleton
+            style="
+                --neutral-fill-rest: #e1dfdd;
```

Comment:
Is this contextual? I see it repeated throughout here. If it's the common pattern, perhaps it should be the default?

Reply:
You're right, it looks like it should be a default. I'll revise.